apps/google-analytics-4/frontend/src/components/config-screen/GoogleAnalyticsConfigPage/GoogleAnalyticsConfigPage.spec.tsx

```diff
@@ -106,15 +121,26 @@ describe('Installed Service Account Key', () => {
       },
     });
     mockSdk.app.isInstalled.mockReturnValue(true);
+    process.on('unhandledRejection', () => {
```

Comment:
This is essentially the equivalent I found for how to ignore that unhandled rejection error the Vitest way, similar to what is in the `setupTests` file for Jest. But this is a bit better since it only swallows that error for this particular test, so we don't risk our test not catching legitimate errors that could crop up in future development.